### PR TITLE
Fixer flag in configuration

### PR DESCRIPTION
**Pull request by prokopsimek**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have added new practice to practice list in README.md.
- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/scanner/Scanner.spec.ts [hunk 1]
```diff
@@ -64,18 +70,78 @@ describe('Scanner', () => {
...
-    containerCtx = createTestContainer({ fix: true });
-    ESLintWithoutErrorsPractice.prototype.fix = fixMock;
-    ESLintWithoutErrorsPractice.prototype.evaluate = () => Promise.resolve(PracticeEvaluationResult.notPracticing);
-    containerCtx.container.bind('ESLintWithoutErrorsPractice').to(ESLintWithoutErrorsPractice);
-    const scanner = containerCtx.container.get(Scanner);
+  describe('fixer', () => {
+    it.only('runs fix when fix flag set to true', async () => {
+      jest.setTimeout(40000);
+      nock.cleanAll();
+      nock.restore();
```
Comment (prokopsimek) [inline src/scanner/Scanner.spec.ts]:
```suggestion
      // nock.restore();
```

Comment (prokopsimek) [reply src/scanner/Scanner.spec.ts]: https://github.com/nock/nock/#restoring

Just `cleanAll()` works fine for me.

File: src/scanner/Scanner.spec.ts [hunk 2]
```diff
@@ -64,18 +66,73 @@ describe('Scanner', () => {
...
-  it('runs fix when fix flag set to true', async () => {
-    jest.setTimeout(20000);
-    const fixMock = jest.fn();
-    containerCtx = createTestContainer({ fix: true });
-    ESLintWithoutErrorsPractice.prototype.fix = fixMock;
-    ESLintWithoutErrorsPractice.prototype.evaluate = () => Promise.resolve(PracticeEvaluationResult.notPracticing);
-    containerCtx.container.bind('ESLintWithoutErrorsPractice').to(ESLintWithoutErrorsPractice);
-    const scanner = containerCtx.container.get(Scanner);
+  describe('fixer', () => {
+    jest.setTimeout(40000);
```
Comment (prokopsimek) [inline src/scanner/Scanner.spec.ts]: Is it necessary to have so big timeout here? I'd remove the timeout.

Comment (vlasy) [reply src/scanner/Scanner.spec.ts]: :wrench:  Fixed in d17c4435d2a8b06518aa39018ebfa3fabd3fbbc2

**Pull request merged**